web/src/integration-test/java/ca/hendriks/bartender/web/functionaltest/ingredient/BddIngredientService.java

```diff
@@ -67,4 +67,8 @@ public void cleanUpRepository(){
         ingredientRepository.deleteAll();
     }
 
+    public void deleteIngredient(final String ingredientName) {
+        final Ingredient ingredientToBeDeleted = ingredientRepository.findByName(ingredientName);
+        ingredientRepository.deleteById(ingredientToBeDeleted.getId());
```

Comment:
hmm... is this just temporary code? it's not the BDD that should do the deleting... but the business logic in `IngredientsRestController`

Reply:
same problem with the method `updateIngredient` as well. I missed it the first time. Use `BddMockMvcService` to fire off the call to `IngredientsRestController`

Reply:
Oops thanks.. was confused over where the 'real' code was